backend/oasst_backend/utils/hugging_face.py

```diff
@@ -41,6 +51,10 @@ async def post(self, input: str) -> Any:
             async with session.post(self.api_url, headers=self.headers, json=payload) as response:
```

Comment:
do we know what happens if the text is longer than the intended text length of the model?

Reply:
I am not sure about this.